espnet2/schedulers/noam_lr.py

```diff
@@ -39,8 +35,6 @@ def __init__(
     def get_lr(self):
         step_num = self.last_epoch + 1
         return [
-            lr
-            * self.model_size ** -0.5
```

Comment:
Does this mean that `lr` would be changed from the previous one by a factor of `self.model_size ** -0.5`?
Does this modification change `lr` tuning range?

Reply:
Sorry, this implementation had a bug, that is not my original intention of PR. The lr will be multiplied by `self.model_size ** -0.5` at every steps.